elasticsearch/templates/statefulset.yaml

```diff
@@ -124,6 +124,9 @@ spec:
       {{- end }}
       containers:
       - name: "{{ template "name" . }}"
+        securityContext:
+          runAsNonRoot: true
+          runAsUser: 1000
```

Comment:
This value will need to be configurable for systems where the running user is not always 1000 (like Openshift). I think it makes the most sense to expose this variable in the values.yaml with these as the defaults.

Reply:
Thanks, I took a similar approach as `kibana` with `podSecurityContext` and `securityContext` in the values file.